Content.Shared/FeedbackSystem/SharedFeedbackSystem.Events.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Linq;
+using Content.Shared.CCVar;
+using Content.Shared.GameTicking;
+using Robust.Shared.Configuration;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.FeedbackSystem;
+
+public sealed partial class SharedFeedbackSystem
+{
+    [Dependency] private readonly SharedFeedbackSystem _feedback = default!;
+    [Dependency] private readonly IPrototypeManager _proto = default!;
+    [Dependency] private readonly IConfigurationManager _confg = default!;
+
+    private List<string> _validOrigins = [];
+
+    public void EventInitialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<RoundEndMessageEvent>(OnRoundEnd);
+
+        _confg.OnValueChanged(CCVars.FeedbackValidOrigins, OnFeedbackOriginsUpdated, true);
```

Comment:
You can use the Subs helper here:
```suggestion
        Subs.CVar(_confg, CCVars.FeedbackValidOrigins, OnFeedbackOriginsUpdated, true);
```
This automatically unsubscres when the entity system gets shut down so it won't leak a subscription.